RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate containers/Avatar to hooks

## Proposed changes


## Issue(s)	


## How to test or reproduce

Make sure your avatars are working properly.
Check if the error resolved in this PR #4042 has not reoccurred.

## Screenshots

## Types of changes




## Checklist



## Further comments
